src/kibit/driver.clj

```diff
@@ -39,10 +39,10 @@
                                         :reporter (name-to-reporter (:reporter options)
                                                                     cli-reporter)
                                         :rules (or rules all-rules))
-                            (catch Exception e
+                            (catch Throwable e
                               (binding [*out* *err*]
-                                (println "Check failed -- skipping rest of file")
-                                (println (.getMessage e))))))
+                                (println (str "At " (.getPath file) ":0:\nCheck failed -- skipping rest of file"))
```

Comment:
Stack trace is meaningless in that case. I've seen it and it doesn't have
any useful information for user to investigate the problem. But I think at
least we could try to track top level form that raised an exception and
print the line number in log.

On Tue, Jun 14, 2016, 5:26 AM Daniel Compton <notifications@github.com>
wrote:

> In src/kibit/driver.clj
> <https://github.com/jonase/kibit/pull/161#discussion_r66880219>:
>
> >                                (binding [*out* *err*]
> > -                                (println "Check failed -- skipping rest of file")
> > -                                (println (.getMessage e))))))
> > +                                (println (str "At " (.getPath file) ":0:\nCheck failed -- skipping rest of file"))
>
> is it possible to get the real line number here? If we just log the stack
> trace, can Emacs understand that?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/jonase/kibit/pull/161/files/4db26119c13ec3780f4596815e6329f3e0e796b8#r66880219>,
> or mute the thread
> <https://github.com/notifications/unsubscribe/ABbhy4YRejLOPvxe9u8bqdO921wAgJhEks5qLdkegaJpZM4I0kH6>
> .
>